bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -177,13 +180,24 @@ export class UIEvents
       sx: x - left
       sy: y - top
     }
+    e.bokeh = extend(e.bokeh, extras)
+    event_cls = PointEvent.event_class(e)
```

Comment:
Why `PointEvent`? I presume this should be `Event` (or `BokehEvent` after changes). The same applies below.